python/lsst/summit/utils/guiders/reading.py

```diff
@@ -763,30 +774,87 @@ def getGuiderRawStamps(self, dayObs: int, seqNum: int) -> dict[str, Stamps]:
                 raise DatasetNotFoundError(f"No data for {detName} on {dayObs=} {seqNum=}") from e
         return rawStamps
 
-    def subtractMedianRowBias(self, rawStampsDict: dict[str, Stamps]) -> dict[str, Stamps]:
+    def processStamps(
+        self,
+        rawStampsDict: dict[str, Stamps],
+        doSubtractMedian: bool = True,
+        columnMaskK: float = 50.0,
```

Comment:
Remove these defaults completely, make them required args so you can't set them in two places.